blkio.go

```diff
@@ -181,9 +181,6 @@ func (b *blkioController) readEntry(devices map[deviceKey]string, path, name str
 	defer f.Close()
 	sc := bufio.NewScanner(f)
 	for sc.Scan() {
-		if err := sc.Err(); err != nil {
-			return err
-		}
```

Comment:
I didn't get, so you suggest to ignore scan errors and continue to try to collect data from broken scan? Ho should it work?

Reply:
Please see https://golang.org/pkg/bufio/#Scanner.Scan documentation. In case of an error, Scan() returns false, and the Err() should be checked right after the  `for s.Scan()` loop.